cmd/oci-create-layer/oci-create-layer.1.md

```diff
@@ -0,0 +1,29 @@
+% OCI(1) OCI-CREATE-LAYER User Manuals
+% OCI Community
+% October 2016
+# NAME
+oci-create-layer \- Create filesystem changeset
+
+# SYNOPSIS
+**oci-create-layer** [child] [parent] [flags]
+
+# DESCRIPTION
+`oci-create-layer` creates a filesystem changeset from two layers. It compares child with parent and generates a filsystem diff, pack the diff into a uncompressed tar archive. The output tar archive name is the child name with .tar suffix by default, use `--dest` to specify a custom one.
```

Comment:
Yeah, that sounds good to me.  Although I think `--output` is a more popular name for this sort of thing than `--dest`.